source/js/fields.js

```diff
@@ -127,6 +130,22 @@ class Fields {
         }
     }
 
+    imageDrop(formInput) {
+        const imagePreviewId = formInput.parentNode.getAttribute('data-image-preview');
+        const imagePreviewArea = document.getElementById(imagePreviewId);
+        imagePreviewArea.addEventListener('dragover', (e) => {
```

Comment:
does this work even when/if imagePreviewArea is undefined?

Reply:
That's a fair point, this method should never run unless `data-image-preview` is defined on the input field. In which case imagePreviewArea _should_ be defined but I suppose a fail safe would be good to add